src/scrapers/mizrahi.ts

```diff
@@ -66,7 +66,7 @@ function createLoginFields(credentials: ScraperCredentials) {
 
 function getPossibleLoginResults(page: Page): PossibleLoginResults {
   return {
-    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL],
+    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL, async () => !!(await page.$x('//title[contains(., "עדכון כתובת דוא\'\'ל")]'))],
```

Comment:
Would add an XPath expression generation helper to increase readability and ease of use for cases when there are several items within a contains clause for example.
Not in the scope of this PR though.